lib/bitters/generator.rb

```diff
@@ -1,37 +1,43 @@
-require 'bitters/version'
-require 'fileutils'
-require 'pathname'
-require 'thor'
+require "bitters/version"
+require "fileutils"
+require "pathname"
+require "thor"
 
 module Bitters
   class Generator < Thor
-    map ['-v', '--version'] => :version
-    map ['scaffold'] => :install
-    map ['delete'] => :remove
+    map ["-v", "--version"] => :version
+    map ["scaffold"] => :install
+    map ["delete"] => :remove
 
-    desc 'install', 'Install Bitters into your project'
-    method_options path: :string
+    desc "install", "Install Bitters into your project"
+    method_options path: :string, syntax: :string
     def install
       if bitters_files_already_exist?
         puts "Bitters files already installed, doing nothing."
       else
         install_files
-        puts "Bitters files installed to #{install_path}/base"
+        puts "Bitters files installed to #{install_path}/base."
+        if requested_sass?
+          puts "Converted files to Sass syntax."
+        end
```

Comment:
Sorry about the note down on "reset", I meant to put that here. This output should move down to `convert_syntax_to_sass` after the `Dir.glob` end. With it being here, I was getting the "Could not convert..." message followed by "Converted files..."